### ISSUE TITLE
demo

### ISSUE THREAD

**Issue by manzezs** [collective/demo.plone.de#24]
hi the demo is not in English after i press the flag in top right

**Comment by pbauer** [comment]
@manzezs thanks for the report. I just tried it and it seems to work fine (FF, Chrome and Safari). 
Did you try the mobile view by change? Whan the screen is very narrow the searchbox makes the language-selector unuseable. I'll create a ticket for that.

**Comment by manzezs** [comment]
No I did not I used my laptop.


Sent from Samsung Mobile


-------- Original message --------
From: Philip Bauer
Date:08/03/2018 2:18 PM (GMT+00:00)
To: "collective/demo.plone.de"
Cc: manzezs , Mention
Subject: Re: [collective/demo.plone.de] demo (#24)


@manzezs<https://github.com/manzezs> thanks for the report. I just tried it and it seems to work fine (FF, Chrome and Safari).
Did you try the mobile view by change? Whan the screen is very narrow the searchbox makes the language-selector unuseable. I'll create a ticket for that.
